src/conf.py

```diff
@@ -20,23 +20,14 @@
 read_the_docs_build = os.environ.get('READTHEDOCS', None) == 'True'
 if read_the_docs_build:
 
-    # Makes sure directory exists for doxygen output
-    cwd=os.getcwd()
-    buildpath=os.path.join(cwd,"_build")
-    if (os.path.isdir(buildpath) == 0):
-        os.mkdir(buildpath)
-    htmlpath=os.path.join(buildpath,"html")
-    if (os.path.isdir(htmlpath) == 0):
-        os.mkdir(htmlpath)
-
     # Modify Doxyfile for ReadTheDocs compatibility
     with open('./docs/doxygen/Doxyfile.in', 'r') as f:
         fdata = f.read()
     fdata = fdata.replace('@PROJECT_SOURCE_DIR@', '.')
     with open('./docs/doxygen/Doxyfile.in', 'w') as f:
         f.write(fdata)
     with open('./docs/doxygen/Doxyfile.in', 'a') as f:
-        f.write("\nOUTPUT_DIRECTORY=./_build/html/doxygen")
+        f.write("\nOUTPUT_DIRECTORY=../_readthedocs/html/doxygen")
```

Comment:
@rhornung67 

This PR by ReadtheDocs introduced the change, and will likely impact most github projects using doxygen on ReadTheDocs:
readthedocs/readthedocs.org#9888

Doxygen generation broken in latest RAJA:
https://raja.readthedocs.io/en/develop/doxygen/html/index.html

The breaking change in their PR was renaming the hosted output directory from `./_build/html/...` to  `./_readthedocs/html/...`

Seems like alot of projects are dependent on this `_build` directory:
https://github.com/search?q=filename%3Aconf.py+doxygen+%26%26+_build&type=Code

Reply:
@bmhan12 Thanks for pointing this out.